demo/portal/src/app/config.js

```diff
@@ -14,7 +14,7 @@ var angular = require('angular');
  * @constructor
  */
 function AppConfig($stateProvider, $urlRouterProvider) {
-  $urlRouterProvider.otherwise('/workorders/list');
+  $urlRouterProvider.otherwise('/unauthorised');
```

Comment:
[Question] This field was used to redirect to /workorders view when nothing was provided.
Is that change essential ?